Confus/Confus/Maze.cpp

```diff
@@ -3,11 +3,12 @@
 
 namespace Confus
 {
-	Maze::Maze(irr::IrrlichtDevice* a_Device, irr::core::vector3df a_StartPosition, bool a_NeedRender)
-		:m_MazeSizeX(60), m_MazeSizeY(60)
+	Maze::Maze(irr::IrrlichtDevice* a_Device, int a_MazeSizeX, int a_MazeSizeY, irr::core::vector3df a_StartPosition,float a_PathScalar, bool a_NeedRender)
```

Comment:
Any reason to not use something like irr::dimension(s) (or whatever it is) over individual values?